src/rlgl.h

```diff
@@ -2018,28 +2011,32 @@ void rlLoadExtensions(void *loader)
     for (int i = 0; i < numExt; i++) TRACELOG(RL_LOG_INFO, "    %s", glGetStringi(GL_EXTENSIONS, i));
 #endif
 
+#if defined(GRAPHICS_API_OPENGL_21)
```

Comment:
This code structure looks a bit complex. I prefer to just use a `#if defined(GRAPHICS_API_OPENGL_21)` check for the required extensions and duplicate code, it's more visual.